migrate/troubleshooting.md

```diff
@@ -8,7 +8,144 @@ tags: [recovery, logical backup, pg_dump, pg_restore]
+
+
+## Why does live migration log "no tuple identifier" warning?
+
+Live migration logs a warning `WARNING: no tuple identifier for UPDATE in table`
+when it cannot determine which specific rows should be updated after receiving an
+`UPDATE` statement from the source database during replication. This occurs when tables
+in the source database that receive `UPDATE` statements lack either a `PRIMARY KEY` or
+a `REPLICA IDENTITY` setting. For live migration to successfully replicate `UPDATE` and
+`DELETE` statements, tables must have either a `PRIMARY KEY` or `REPLICA IDENTITY` set
+as a prerequisite.
+
+
+## Set REPLICA IDENTITY on PostgreSQL partitioned tables
+
+If your PostgreSQL tables use native partitioning, setting `REPLICA IDENTITY` on the
+root (parent) table will not automatically apply it to the partitioned child tables.
+You must manually set `REPLICA IDENTITY` on each partitioned child table.
+
+
+## Can I use read/failover replicas as source database for live migration?
+
+Live migration does not support replication from read or failover replicas. You must
+provide a connection string that points directly to your source database for
+live migration.
+
+
+## Can I use live migration with a Postgres connection pooler like PgBouncer?
+
+Live migration does not support connection poolers. You must provide a
+connection string that points directly to your source and target databases
+for live migration to work smoothly.
+
+
+## Can I use Timescale Cloud instance as source for live migration?
+
+No, Timescale Cloud cannot be used as a source database for live migration.
+
+
+## How can I exclude a schema/table from being replicated in live migration?
+
+At present, live migration does not allow for excluding schemas or tables from
+replication, but this feature is expected to be added in future releases.
+However, a workaround is available for skipping table data using the `--skip-table-data` flag.
+For more information, please refer to the help text under the `migrate` subcommand.
+
+
+## "target_replay_rate" is 0B/s even after waiting for several minutes
+
+This is a known issue with live migration. In some cases, live migration may log the
+following progress information.
+
+```sh
+WARNING: live-replay not keeping up with source load (source_wal_rate: 2.0MiB/s, target_replay_rate: 0B/s, replay_lag: 288MiB)
+```
+
+This means that live migration is not replicating changes to the target, causing the
+replay lag to keep increasing. If you see the `target_replay_rate` at `0B/s` for more
+than 10 minutes, a workaround is to stop the live migration using `CTRL + C` and
+resume it using the resume flag: `./live-migration migrate --resume`.
+This will restart the `migrate` process from where it left, and the `target_replay_rate` should no
+longer be `0B/s`. If the issue persists, please reach out to the Timescale migration team
+or share it in the `#migration` channel on Timescale community Slack.
+
```

Comment:
This is no longer required. We already fixed it on v0.0.22.

Reply:
Removed